Src/AutoFixture/Dsl/NodeComposer.cs

```diff
@@ -411,7 +412,7 @@ public IPostprocessComposer<T> WithAutoProperties()
             Expression<Func<T, TProperty>> propertyPicker)
         {
             var m = propertyPicker.GetWritableMember().Member;
-            if (m.ReflectedType != typeof(T))
+            if (m.DeclaringType != typeof(T))

```

Comment:
I thought so too :) Anyway, I can fix any issues that might appear, but the tests seem to cover a lot.